pkg/openevec/config.go

```diff
@@ -37,10 +44,12 @@ type EdenConfig struct {
 	EdenBin      string `mapstructure:"eden-bin"`
 	TestBin      string `mapstructure:"test-bin"`
 	TestScenario string `mapstructure:"test-scenario"`
+	Tests        string `mapstructure:"tests"`
```

Comment:
is it used anywhere?

Reply:
Used in escript, won't need this once we remove it. Then we can just see if the field is used and remove it, where needed